eq-author-api/db/models/DynamoDB.js

```diff
@@ -0,0 +1,93 @@
+const dynamoose = require("dynamoose");
+
+let throughput = "ON_DEMAND";
+let questionnanaireTableName = "author-questionnaires";
+let questionnanaireVersionsTableName = "author-questionnaire-versions";
+
+if (process.env.DYNAMO_ENDPOINT !== "") {
+  dynamoose.local(process.env.DYNAMO_ENDPOINT);
+  throughput = { read: 10, write: 10 }; // DynamoDB local doesn't yet support on-demand
+}
+
+if (process.env.DYNAMO_QUESTIONNAIRE_TABLE_NAME !== "") {
+  questionnanaireTableName = process.env.DYNAMO_QUESTIONNAIRE_TABLE_NAME;
+}
+
+if (process.env.DYNAMO_QUESTIONNAIRE_VERSION_TABLE_NAME !== "") {
+  questionnanaireVersionsTableName =
+    process.env.DYNAMO_QUESTIONNAIRE_VERSION_TABLE_NAME;
+}
+
+const Schema = dynamoose.Schema;
```

Comment:
This var seems pointless. You can go `new dynamoose.Schema` without losing anything.